packages/expo-modules-core/ios/Core/AppContext.swift

```diff
@@ -158,6 +163,10 @@ public final class AppContext: NSObject, @unchecked Sendable {
   // MARK: - UI
 
   public func findView<ViewType>(withTag viewTag: Int, ofType type: ViewType.Type) -> ViewType? {
+    if let view = hostWrapper?.findView(withTag: viewTag) as? ViewType {
+      return view
+    }
+    
     return reactBridge?.uiManager.view(forReactTag: NSNumber(value: viewTag)) as? ViewType
```

Comment:
do we keep this compatible call path?

if not (since we don't support the legacy arch anymore), maybe we can just bind RCTHost directly into AppContext?